include/eblob/blob.h

```diff
@@ -750,6 +750,11 @@ enum eblob_stat_global_flavour {
 	EBLOB_GST_DATASORT_VIEW_USED_NUMBER,
 	EBLOB_GST_DATASORT_SORTED_VIEW_USED_NUMBER,
 	EBLOB_GST_DATASORT_SINGLE_PASS_VIEW_USED_NUMBER,
+	EBLOB_GST_DATASORT_NUMBER_BLOBS,
```

Comment:
Let's stick here and below to existing naming pattern, e.g. `EBLOB_GST_DATASORT_BLOBS_NUMBER`

Reply:
This naming pattern looks not correct.

Reply:
`NUMBER_BLOBS` is incorrect either, it should by `NUMBER_OF_BLOBS` but I suggest sticking to consistent naming. IMO better naming are `...blobs_amount`, `...blobs_count`, but consistency is in the first place here.

Reply:
done